src/cpu/core_dyn_x86/cache.h

```diff
@@ -172,7 +176,7 @@ class CodePageHandler : public PageHandler {
 			}
 			memset(invalidation_map,0,4096);
 		}
-		(*(Bit32u*)&invalidation_map[addr])+=0x1010101;
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).